src/Orchard.Cms.Web/Modules/Orchard.Setup/Views/Setup/Index.cshtml

```diff
@@ -106,6 +107,7 @@
                 <option value="@provider.Value" data-connection-string="@provider.HasConnectionString">@provider.Name</option>
                 }
             </select>
+            <span asp-validation-for="DatabaseProvider" class="text-danger"></span>
```

Comment:
May be we can find a way to have only one <span asp-validation-for="DatabaseProvider" class="text-danger"></span>